scripts/hudson/narayana.sh

```diff
@@ -392,7 +393,7 @@ function clone_as {
     git reset --hard jbosstm/5_BRANCH
     [ $? -eq 0 ] || fatal "git reset 5_BRANCH failed"
     git clean -f -d -x
-    [ $? -gt 1 ] || fatal "git clean failed"
+    [ $? -ge 0 ] || fatal "git clean failed"
```

Comment:
Shouldn't this be `[ $? -ne 0 ] && fatal "git clean failed"` or `[ $? -eq 0 ] || fatal "git clean failed"`

Reply:
I think I probably erred on caution but realised that it was a 0. Anyway, I will pull this commit out for now and put it in a different PR

Reply:
I didn't mind it being fixed here since it was a tiny change, but if you want do in a separate PR then that's okay too even though it will consume CI resources (since you would need to run one of the AS_TESTS, XTS or RTS axes).